src/sentry/api/endpoints/organization_events.py

```diff
@@ -126,3 +127,46 @@ def data_fn(offset, limit):
                         request, organization, params["project_id"], results
                     ),
                 )
+
+
+class OrganizationEventsGeoEndpoint(OrganizationEventsV2Endpoint):
+    def get(self, request, organization):
+        if not self.has_feature(organization, request):
+            return Response(status=404)
+
+        try:
+            params = self.get_snuba_params(request, organization)
+        except NoProjects:
+            return Response([])
+
+        maybe_aggregate = request.GET.get("field")
+
+        if not maybe_aggregate:
+            return Response({"detail": ["No column selected"]}, status=400)
```

Comment:
```suggestion
            raise ParseError(detail="No column selected")
```

You'd also need `from rest_framework.exceptions import ParseError` in the import list.